src/main/java/com/purbon/kafka/topology/actions/topics/SyncTopicAction.java

```diff
@@ -55,19 +56,21 @@ public void syncTopic(Topic topic, String fullTopicName, Set<String> listOfTopic
     }
 
     if (topic.getSchemas().isPresent()) {
-      final TopicSchemas schemas = topic.getSchemas().get();
+      // changes to support multiple schemas: Issue 155
+      final List<TopicSchemas> schemas = topic.getSchemas().get();
+      for (TopicSchemas schema : schemas) {
+        schema
+            .getKeySchemaFile()
+            .ifPresent(
+                keySchemaFile ->
+                    schemaRegistryManager.register(fullTopicName + "-key", keySchemaFile));
```

Comment:
Some early feedback: when registering the schema, I understand we will need to take into account the subject name strategy (for key and value) to decide the actual subject name, right? I mean, something like:
* TopicNameStrategy: `<topic>-key`, `<topic>-value` .   (as done above, by default)
* RecordNameStrategy: `<record-type>` .   (the avro record type int he schema, regardless of topic)
* TopicRecordNameStrategy: `<topic>-<record-type>`

Reply:
@m-dias thanks. Would you see the need for having multiple naming strategies per topic? I have cannot think of a situation where would you want to mix in a single topic as this would create multiple I believe? @purbon

Reply:
We would set TopicRecordNameStrategy as a property value that we pass to the KafkaConsumer.  So I think that would require it  to be on a per topic basis.

Reply:
TopicRecordNameStrategy is something you set on the serializer, however from the KTB we need to know this at the topic level. Probably thinking under the schema section. Based on the value there, as @m-dias said, this will select the name registered by the SchemaManager.